src/code_8048480.c

```diff
@@ -294,3 +348,433 @@ void sub_80487CC(struct Entity *pokemon, struct Entity * target, u32 param_3, u3
 {
     sub_8078B5C(pokemon, target, param_3, param_4, 1);
 }
+
+static inline bool8 sub_80487E0_sub(struct EntityInfo *pokemonInfo)
```

Comment:
This function is named `JoinLocationCannotUseItems` in other places where it appears, so the numerous occurrences in this file can also use that name. Add an ordinal like `JoinLocationCannotUseItems_1` to avoid name conflicts.